drivers/ethernet/eth_native_posix.c

```diff
@@ -587,3 +595,22 @@ LISTIFY(CONFIG_ETH_NATIVE_POSIX_INTERFACE_COUNT, PTP_INIT_FUNC, (), _)
 LISTIFY(CONFIG_ETH_NATIVE_POSIX_INTERFACE_COUNT, DEFINE_PTP_DEVICE, (;), _);
 
 #endif /* CONFIG_ETH_NATIVE_POSIX_PTP_CLOCK */
+
+static void add_native_posix_options(void)
+{
+	static struct args_struct_t eth_native_posix_options[] = {
+		{
+			.is_mandatory = false,
+			.option = "eth-if",
+			.name = "name",
+			.type = 's',
+			.dest = (void *)&if_name_cmd_opt,
+			.descript = "Name of the eth interface to use",
```

Comment:
Just a minor usability improvement sugesstion. Fine otherwise
```suggestion
			.descript = "Name of the eth interface to use. By default '"
                                       CONFIG_ETH_NATIVE_POSIX_DRV_NAME "'",
```